sdk/common/src/main/java/io/opentelemetry/sdk/common/export/RetryPolicy.java

```diff
@@ -66,6 +72,9 @@ public static RetryPolicyBuilder builder() {
   /** Returns the backoff multiplier. */
   public abstract double getBackoffMultiplier();
 
+  /** Returns the predicate if exception is retryable. */
+  public abstract Predicate<IOException> getRetryExceptionPredicate();
```

Comment:
We can't have a default retry predicate because the default predicate varies by sender implementation. Instead, let's make this method `@Nullable`, and use `null` to indicate that the "protocol and sender specific default retry predicate should be used". 

I sketched out a commit of how this works: https://github.com/open-telemetry/opentelemetry-java/commit/317397a6f5bd9e387143aeaaa4c57a8d284995e2

Notice how JdkHttpSender and OkHttpSender each have their own default predicate. Because the RetryPolicy is more generic than JdkHttpSender, OkHttpSender, and even more generic than OTLP, it can't contain protocol and sender specific logic.

```suggestion
  @Nullable
  public abstract Predicate<IOException> getRetryExceptionPredicate();
```

Reply:
done